app/templates/layouts/_base.html

```diff
@@ -1,41 +1,41 @@
-<!doctype html>
-{% set cdn_hash = "v2.0.0" %}
-{% set cdn_url_prefix = "https://cdn.ons.gov.uk/sdc/"~cdn_hash %}
-{% import 'macros/helpers.html' as helpers %}
+<!DOCTYPE html>
+{% set cdn_hash = "v2.0.0" %} {% set cdn_url_prefix = "https://cdn.ons.gov.uk/sdc/"~cdn_hash %} {% import 'macros/helpers.html' as helpers %}
 <!--[if lt IE 7]>      <html lang="en-gb" dir="ltr" class="no-js lt-ie10 lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
 <!--[if IE 7]>         <html lang="en-gb" dir="ltr" class="no-js lt-ie10 lt-ie9 lt-ie8"> <![endif]-->
 <!--[if IE 8]>         <html lang="en-gb" dir="ltr" class="no-js lt-ie10 lt-ie9"> <![endif]-->
 <!--[if IE 9]>         <html lang="en-gb" dir="ltr" class="no-js lt-ie10"> <![endif]-->
-<!--[if gt IE 9]><!--> <html lang="en-gb" dir="ltr" class="no-js"> <!--<![endif]-->
+<!--[if gt IE 9]><!-->
+<html lang="en-gb" dir="ltr" class="no-js">
+  <!--<![endif]-->
   <head>
     {%- block head %}
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
-    <title>{% block page_title %}{{ _('ONS Survey') }}{% endblock page_title %}</title>
-    <meta content="" name="description">
-    <meta content="width=device-width, initial-scale=1" name="viewport">
-    <link rel="apple-touch-icon" sizes="180x180" href="{{ cdn_url_prefix }}/favicons/apple-touch-icon.png">
-    <link rel="icon" type="image/png" href="{{ cdn_url_prefix }}/favicons/favicon-32x32.png" sizes="32x32">
-    <link rel="icon" type="image/png" href="{{ cdn_url_prefix }}/favicons/favicon-16x16.png" sizes="16x16">
-    <link rel="manifest" href="{{ cdn_url_prefix }}/favicons/manifest.json">
-    <link rel="mask-icon" href="{{ cdn_url_prefix }}/favicons/safari-pinned-tab.svg" color="#5bbad5">
-    <meta name="theme-color" content="#ffffff">
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>{% block page_title %}{{ _("ONS Survey") }}{% endblock page_title %}</title>
+    <meta content="" name="description" />
+    <meta content="width=device-width, initial-scale=1" name="viewport" />
+    <link rel="apple-touch-icon" sizes="180x180" href="{{ cdn_url_prefix }}/favicons/apple-touch-icon.png" />
+    <link rel="icon" type="image/png" href="{{ cdn_url_prefix }}/favicons/favicon-32x32.png" sizes="32x32" />
+    <link rel="icon" type="image/png" href="{{ cdn_url_prefix }}/favicons/favicon-16x16.png" sizes="16x16" />
+    <link rel="manifest" href="{{ cdn_url_prefix }}/favicons/manifest.json" />
+    <link rel="mask-icon" href="{{ cdn_url_prefix }}/favicons/safari-pinned-tab.svg" color="#5bbad5" />
+    <meta name="theme-color" content="#ffffff" />
```

Comment:
Can you revert these changes ☝️ I think they were introduced by your auto-formatting settings, but it just makes it more difficult to see what the actual change was.